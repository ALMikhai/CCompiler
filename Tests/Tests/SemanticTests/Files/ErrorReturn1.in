struct a {} a1;
struct a f() {
    return 1;
}