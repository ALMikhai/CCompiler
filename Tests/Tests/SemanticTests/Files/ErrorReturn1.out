(3:12): semantic error: incompatible types when returning type 'const INT' but 'STRUCT called a' was expected
